src/commands/registry.rs

```diff
@@ -126,6 +134,9 @@ pub struct Login {
     )]
     pub password_stdin: bool,
 
+    /// Container registry server
```

Comment:
We (again generally and inconsistently) tend to avoid "container" in this context, because Spin OCI images are not containers and we don't want to risk ambiguity on that.  If we qualify "registry" then we prefer "OCI" as in the specification (e.g. in the `spin registry` root subcommand help text).  This is all a very long explanation of why I suggest 

```suggestion
    /// OCI registry server (e.g. ghcr.io)
```

(again with the merging paragraphs)